3.7/root/opt/app-root/etc/scl_enable

```diff
@@ -4,6 +4,6 @@
 unset BASH_ENV PROMPT_COMMAND ENV
 if head "/etc/redhat-release" | grep -q "^CentOS Linux release 7" || \
    head "/etc/redhat-release" | grep -q "^Red Hat Enterprise Linux\( Server\)\? release 7"; then
-    source scl_source enable httpd24 $NODEJS_SCL rh-python36
+    source scl_source enable httpd24 $NODEJS_SCL 
```

Comment:
Even though we don't need it right now, I would probably add the rh-python37 definition into the spec, so this doesn't bite us later.

Reply:
Fixed in two fixup commits - one for multispec and one for generated content.